tests/test_forms.py

```diff
@@ -158,3 +169,62 @@ def test_silently_passes_if_csrf_token_not_present(self):
         data = {'key': 'value', 'key2': 'value2'}
         cleaned_data = remove_csrf_token(data)
         assert cleaned_data == data
+
+
+class TestFormatWTFormErrors:
+    def test_empty_list_if_no_errors(self, app):
+        with app.app_context():
```

Comment:
So annoying that we need this `app_context` for a whole bunch of utils tests 😿 Not something to fix here, anyway.

Reply:
I can probably make it less bad. But yeah, it's still annoying. :(

Reply:
Mmm, nope, I can't really make it less bad.

I'll make a tech debt ticket :( Hopefully I'll get some time to pick it up later.